js/client/src/components/app/Graph.js

```diff
@@ -14,20 +14,42 @@ const Graph = ({ polygonData, darkMode }) => {
   const [title, setTitle] = useState({ value: 'make', label: 'Make of Vehicle' });
 
   const fetchGraph = async () => {
-    const response = await axios
-      .get(`${API_URL}/api/citation/graph`, {
-        params: {
-          polygon: polygonData,
-          filterBy: selectedKey,
-        },
+    try {
+      if (Array.isArray(polygonData)) {
+        const response = await axios
+          .get(`${API_URL}/api/citation/graph`, {
+            params: {
+              polygon: polygonData,
+              filterBy: selectedKey,
+            },
+          })
+          
+        var parsed = response.data.map((obj) => {
+          obj["y"] = (parseInt(obj.y))
+          return obj
+        })
+
```

Comment:
it's not a performance suggestion, but it looks this part can be moved out of this if/else block, since it shared the same code in both blocks, and also it's good to change var to const.